templates/base.html

```diff
@@ -17,14 +17,15 @@
         <div class="full-page">
             <!-- Nav -->
             <ul class="nav">
-                <li><a class="active" href="{% url 'home' %}">Home</a></li>
-                <li><a class="active" href="{% url 'home' %}#how">How It Works</a></li>
-                <li><a class="active" href="{% url 'artists' %}">Artists</a></li>
+                    <li><a class="active" href="{% url 'home' %}">HOME</a></li>
+                    <li><a class="active" href="{% url 'artists' %}">ARTISTS</a></li>
+<!--                 <li><a class="active" href="{% url 'home' %}#how">How It Works</a></li> -->
```

Comment:
Did we want to take out How It Works from the nav? If so, let's just delete this line entirely.